webplugin/js/app/mck-sidebox-1.0.js

```
@@ -2858,23 +2806,12 @@ const firstVisibleMsg = {
                             );
                             var loginErrorMessage = invalidPasswordLabel;
                             ensureChatLoginModalExists();
-                            if (!AUTH_SUBMIT_TRIGGERED && !isDashboardAuth) {
+                            if (!AUTH_SUBMIT_TRIGGERED) {
                                 _this.resetPreChatLoginError && _this.resetPreChatLoginError();
-                                ensureWidgetIframeVisible();
-                                typeof openWidgetIframe === 'function' && openWidgetIframe();
-                                var kmChatLoginModal = document.getElementById(
-                                    'km-chat-login-modal'
-                                );
+                                var kmChatLoginModal = reopenAuthModal();
```

Biome (2.4.6)

[error] Shouldn't redeclare 'kmChatLoginModal'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)